src/encoded/schemas/donor.json

```diff
@@ -64,9 +64,9 @@
         }
     },
     "boost_values": {
+        "accession": 1.0,
         "external_ids": 1.0,
         "aliases": 1.0,
-        "accession": 1.0,
         "alternate_accessions": 1.0
     }
```

Comment:
if there was no change I am not sure why the 'accession' moved form line 69 to 67?

Reply:
Fixed this. "accession" was originally line 69, but I moved it to line 67 because I thought maybe order had something to do with priority. Obviously doesn't, and now it is moved back to line 69.